src/components/Navbar/Navbar.tsx

```diff
@@ -1,34 +1,34 @@
-import React, { useEffect } from 'react';
-import { Link } from 'react-router-dom';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Link, useLocation } from 'react-router-dom';
 import Logo from '../../assets/logo.svg';
-import useSticky from '../../common/hooks/useSticky';
 import { authRoutes } from '../../routes';
+import { AppState } from '../../store/app/types';
+import { State } from '../../store/types';
 import { useNavbarStyles } from './Navbar.styles';
-import { useLocation } from 'react-router-dom';
 
 export const Navbar: React.FC = () => {
   const classes = useNavbarStyles();
   const location = useLocation();
-  const { sticky, home, locationUpdate } = useSticky();
+  const isHomePage = authRoutes.home === location.pathname;
+  const { isNavSticky } = useSelector<State, AppState>(({ app }) => app);
 
-  useEffect(() => {
-    locationUpdate();
-  }, [location]);
+  const baseNavbarClass = isNavSticky
```

Comment:
I moved this ternary tree out of the template

Reply:
See i wasn't sure if the life cycle would updated the value if it wasn't in the template, but since in this case it's just needed one and done then tots like this.